### DISCUSSION TITLE
add a file "data.json" to website using vercel Rest Api

### DISCUSSION THREAD

**Discussion by sahildhanawade123** [vercel/community#1464] [Help] [answered]
### Summary

i am doing a deployment of github repository using vercel Rest Api to vercel and now i want to add a additional file "data.json" to it using vercel Rest Api show me a example of how to do it.
i don't want "data.json" file to add to github repository. that's why i want to do it some how using vercel Rest Api

**Comment by amyegan** [comment, marked as answer]
Hi @sahildhanawade123! You can [create a new deployment with Vercel REST API](https://vercel.com/docs/rest-api#endpoints/deployments/create-a-new-deployment) or [deploy with Vercel CLI](https://vercel.com/docs/cli/deploying-from-cli#). Both of these options allow you to bypass git
